server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/metrics/GeoBoundsIT.java

```diff
@@ -43,15 +44,15 @@ public void testSingleValuedField() throws Exception {
 
         assertSearchResponse(response);
 
-        GeoBounds geoBounds = response.getAggregations().get(aggName);
+        SpatialBounds<GeoPoint> geoBounds = response.getAggregations().get(aggName);
```

Comment:
Can be use here GeoBounds instead?

Reply:
I was in the process of refactoring. I'll finish the job and then you can decide.